ReSharper.FSharp/test/src/FSharp.Tests/QuickFixes/ReplaceLambdaWithInnerExpressionTest.fs

```diff
@@ -25,6 +25,7 @@ type ReplaceLambdaWithInnerExpressionAvailabilityTest() =
 
     [<Test>] member x.``Text - Replace lambda with reference``() = x.DoNamedTest()
     [<Test>] member x.``Text - Replace lambda with qualified reference``() = x.DoNamedTest()
+    [<Test>] member x.``Text - Replace lambda with reference partial application``() = x.DoNamedTest()
```

Comment:
The names seem overcomplicated. We already know these tests are about lambdas, don't we?
```suggestion
    [<Test>] member x.``Text - Replace with reference partial application``() = x.DoNamedTest()
```